ONSdigital/ras-rm-auth-service
# Introduce-CodeQL workflow file

# What and why?
This PR introduces the codeql code scanning to the auth repo

Also bumps a vulnerability in gevent
# How to test?
Check the workflow files look correct and code scanning is active in the repo within the security tab. 

Deploy and test the app after the gevent version bump. 
# Jira
https://jira.ons.gov.uk/browse/RAS-1355
